### PR TITLE
fix(cx-14211): use property name as ariaLabel

### PR DESCRIPTION
**Pull request by neelabhnagaich**
currently we are passing aria-label with the same semantic name as 'aria-label' due to which screen reader is announcing at md-combobox level which is not expected hence changing it to not use the same pre defined name for property.

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by vinitkumargoel — DISMISSED** [review]

File: web-components/src/components/combobox/ComboBox.ts [hunk 1]
```diff
@@ -1557,7 +1558,6 @@ export namespace ComboBox {
       return html`
         <div
           part="combobox"
-          aria-label=${this.ariaLabelForComboBox}
```
Comment (niti-owner) [inline web-components/src/components/combobox/ComboBox.ts:1560]: Why this is removed? 
Just verify if there is no impact on below PR (CX-10274: Passing List Items Count To The Parent Fix)?
https://github.com/momentum-design/momentum-ui/pull/1561

File: web-components/src/[sandbox]/examples/combobox.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ export const comboBoxTemplate = html`
     style="width: 220px; display: inline-block;"
     .options=${comboBoxOptions}
     placeholder="Placeholder"
-    aria-label="Select the country"
+    ariaLabel="Select the country"
     search-result-aria-label="Select the country, {{count}} results found."
   ></md-combobox>
   <h3>Default with trim space</h3>
```
Comment (niti-owner) [inline web-components/src/[sandbox]/examples/combobox.ts:31]: Can you also add one example with comboboxId ?

File: web-components/src/components/combobox/ComboBox.ts (hunk 1 above)
Comment (neelabhnagaich) [reply web-components/src/components/combobox/ComboBox.ts:1560]: this was extra label on the parent container, which unnecessarily added, hence removed

File: web-components/src/[sandbox]/examples/combobox.ts (hunk 2 above)
Comment (neelabhnagaich) [reply web-components/src/[sandbox]/examples/combobox.ts:44]: done

**Review by merajcal — APPROVED** [review]

**Review by niti-owner — APPROVED** [review]

**Review by merajcal — APPROVED** [review]

**Pull request merged**